usr/src/stand/lib/libfdt/Makefile

```diff
@@ -26,17 +26,15 @@
 
 LIBRARY = libfdt.a
 OBJECTS = \
-	  fdt.o fdt_ro.o fdt_strerror.o fdt_wip.o fdt_empty_tree.o \
-	  fdt_rw.o fdt_sw.o fdt_addresses.o
+	fdt.o fdt_ro.o fdt_strerror.o fdt_wip.o fdt_empty_tree.o \
```

Comment:
We're missing objects from this list (we always were, I must have missed them initially or when I updated things?)  we should have `fdt_overlay.o` in this list I think, and `fdt_wip.o`?

compare to the sources I guess, I'm struggling to think of any reason we could have missed things on purpose.

The same list of objects is in `usr/src/uts/aarch64/Makefile.files` and also looks like it's missing some :\

Reply:
Thank you for your review. I added the missing objects (I kept the changes as a separate commit for now which should help reviewing it).